cpp/service.cc

```diff
@@ -190,3 +225,27 @@ std::deque<chirp::Chirp> ServiceLayer::monitor(const std::string& username,
 
   return found_chirps;
 }
+
+std::deque<chirp::Chirp> ServiceLayer::stream(const std::string& hashtag,
+                                               chirp::Timestamp start) {
+  // TODO: check if username exists
+  std::deque<chirp::Chirp> found_chirps;
+  const std::string hashtag_key = khashtag_ + hashtag;
+  const std::deque<std::string>& hashtag_chirps =
+      store_->get(hashtag_key);
+  for (const std::string& id : hashtag_chirps) {
+    std::deque<std::string> this_chirps_values =
+        store_->get(kchirpValue_ + id);
+    chirp::Chirp thisChirp;
+    if (this_chirps_values.size() > 0) {
+      thisChirp.ParseFromString(this_chirps_values.at(0));
+      int64_t myMicroSeconds = thisChirp.timestamp().useconds();
+      int64_t baslineSeconds = start.useconds();
```

Comment:
is baslineSeconds ever used?

Reply:
You're right. I removed the unused variables in this function.